### ISSUE TITLE
OACMPDM9 Lighting

### ISSUE THREAD

**Issue by The-Gig** [NeonKnightOA/oacmpvol1#19]
I'm checking old "todo lists" from the forum against z_oacmp1v4_beta-220322.pk3.

OACMPDM9: Upper and lower parts of the map are completely dark. Maybe compiled when shaderlist.txt was missing some shader files?
![shot0230](https://user-images.githubusercontent.com/23258514/161799064-aeba1cae-ea31-4c7a-92cd-5ea012fc0d4a.jpg)

**Comment by NeonKnightOA** [comment]
Yeah, lighting is still not my forte. I'll try recompiling this one and dm7 without dark.

**Comment by NeonKnightOA** [comment]
Let me try something here.

**Comment by NeonKnightOA** [comment]
https://mega.nz/file/bMZkRJBa#J_rUm2nz2SpiMFl6kBY0eMceeCLOJ1PLfnz_-CQRzCw
New test version. The map has been more or less relit, but I'm still not happy with the result. Maybe I'll change the skybox's lighting for the next version.

**Comment by NeonKnightOA** [comment]
Lighting problems are still there. I'll relit the map for the next version.

![shot0020](https://user-images.githubusercontent.com/20754036/235353595-8befe827-98f5-43fb-b635-29e2e48e50e3.jpg)

![shot0021](https://user-images.githubusercontent.com/20754036/235353596-7f81f46b-cd7a-46f7-a59c-3f5877d44625.jpg)

![shot0022](https://user-images.githubusercontent.com/20754036/235353597-44d8f442-b0b7-4185-a7b1-7c23ede862ed.jpg)

**Comment by The-Gig** [comment]
Are you sure Akom's shaders were in your shaderlist.txt and thus that the sky shader was emitting light?
